src/main.cpp

```diff
@@ -281,6 +288,15 @@ int main(int argc, char* argv[])
             t.start();
             // wait
             return app.exec();
+        } else if (isSelection) {
```

Comment:
Is this code identical to the isRaw? It seems so to me but I might be missing something. If it is the same lets not duplicate it.